cluster/juju/layers/kubernetes-master/config.yaml

```diff
@@ -26,3 +26,12 @@ options:
     default: "stable"
     description: |
       Snap channel to install Kubernetes master services from
+  api-extra-args:
+    type: string
+    default: ""
+    description: |
+      Space separated list of flags and key=value pairs that will be passed as arguments to
+      kube-apiserver. For example value like this:
+        runtime-config=batch/v2alpha1=true profiling=true
```

Comment:
I'm happy to follow whatever guidelines you have for the k8s layers. If you have an idea for better description let me know I can change it.